rest/clients/pom.xml

```diff
@@ -34,5 +34,6 @@
 
     <modules>
         <module>rest-assured</module>
```

Comment:
It generates a client that wraps around rest-assured making it easier for whomever wants to use it (typically tests). It is a client and it is generic, therefore I think it is at the right place. Like any other clients we shall generate, you ain't forced to use them. You are only required to follow the specs whichever way you want.

Reply:
Follow up question. How do we determine which clients an API repo should generate then? There's like 12. Also, it looks like that one only supports Java 8, so I'm not sure it's a good idea for other repos to depend on it if it's not forward compatible since we would be on the hook for maintaining it.

Reply:
At the end of the day, we don't have to provide any. We do so just to help out but all of them are generated based on the spec. Which one do we provide? The ones we need. I started with the rest assured one by mistake and decide to leave it. I think we only need to put those we think we will be using. Any clients can always go back to the spec and generate their own or simply hardcode their client as long as they adhere to the spec. 

The other benefit here is that in addition to the crude validation done by the OpenAPI parser, it also validates that the generation can be compiled which helped identify caveats in how to write specs. This is a good thing.